setup.py

```diff
@@ -22,81 +22,60 @@ def read(*filenames):
-    import zope.testrunner.options
-    here = os.path.abspath(os.path.join(os.path.dirname(__file__), 'src'))
-    args = sys.argv[:]
-    defaults = ["--test-path", here]
-    options = zope.testrunner.options.get_options(args, defaults)
-    suites = list(zope.testrunner.find.find_suites(options))
-    return unittest.TestSuite(suites)
-
-TESTS_REQUIRE = [
-    'zope.pagetemplate',
-    'zope.testing',
-    'zope.testrunner',
-]
+TESTS_REQUIRE = ["zope.pagetemplate", "zope.testing", "zope.testrunner"]
 
 setup(
-    name='z3c.pt',
-    version='3.1.1.dev0',
-    author='Malthe Borch and the Zope Community',
-    author_email='zope-dev@zope.org',
-    description='Fast ZPT engine.',
+    name="z3c.pt",
+    version="3.1.1.dev0",
+    author="Malthe Borch and the Zope Community",
+    author_email="zope-dev@zope.org",
+    description="Fast ZPT engine.",
     long_description=(
-        read('README.rst')
-        + '\n\n'
-        + read('src', 'z3c', 'pt', 'README.rst')
-        + '\n\n'
-        + read('CHANGES.rst')
+        read("README.rst")
+        + "\n\n"
+        + read("src", "z3c", "pt", "README.rst")
+        + "\n\n"
+        + read("CHANGES.rst")
     ),
-    license='ZPL',
-    keywords='tal tales pagetemplate zope chameleon',
+    license="ZPL",
+    keywords="tal tales pagetemplate zope chameleon",
     classifiers=[
-        'Development Status :: 5 - Production/Stable',
-        'Environment :: Web Environment',
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: Zope Public License',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: Implementation',
-        'Programming Language :: Python :: Implementation :: CPython',
-        'Programming Language :: Python :: Implementation :: PyPy',
-        'Natural Language :: English',
-        'Operating System :: OS Independent',
-        'Topic :: Text Processing :: Markup :: HTML',
-        'Topic :: Text Processing :: Markup :: XML',
-        'Framework :: Zope :: 3',
+        "Development Status :: 5 - Production/Stable",
+        "Environment :: Web Environment",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Zope Public License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: Implementation",
```

Comment:
What's the meaning of this classifier?

Reply:
It was once necessary for PyPI Warehouse as otherwise package with a classifier like `Programming Language :: Python :: Implementation :: CPython` was not found when omitting it.
I checked the search capabilities of PyPI just now and this limitation does not longer seem to exist. So I am going to remove this classifier.